src/core/namespace.c

```diff
@@ -1316,6 +1414,32 @@ int setup_namespace(
                         return log_debug_errno(r, "Failed to decrypt dissected image: %m");
         }
 
+        if (n_mount_paths > 0) {
+                mount_path_loop_devices = new0(LoopDevice *, n_mount_paths);
+                if (!mount_path_loop_devices)
+                        return -ENOMEM;
+
+                for (i = 0; i < n_mount_paths; i++) {
+                        struct stat st;
+
+                        if (stat(mount_paths[i].source, &st) < 0)
+                                return -errno;
+
+                        if (S_ISBLK(st.st_mode)) /* Source is a block device */
+                                mount_path_loop_devices[i] = NULL;
+                        else {
+                                r = loop_device_make_by_path(mount_paths[i].source,
+                                                             mount_paths[i].read_only ? O_RDONLY : O_RDWR,
+                                                             0,
```

Comment:
LO_FLAGS_PARTSCAN?

Reply:
Yes, it's now used in the final implementation that uses dissect_image